docs/development/setup_development.rst

```diff
@@ -20,9 +20,11 @@ Quick Start
 The Docker based environment is suitable for developing the web application
 and updating the documentation.
 
+Ensure Git is installed on your system. Use package management tools to update your local package index. Once the update is complete, you can download and install Git:
```

Comment:
Right now this feels repetitive of the instructions that follow, and only partially (why mention Git and not `make` or Docker itself?). I would suggest we speak to the user's needs at a higher level here, e.g., language like the following (please feel free to reword as you see fit):

> Follow the instructions below to install the requirements for the Docker-based environment for your operating system.